src/components/Loader.js

```diff
@@ -53,7 +55,7 @@ class Loader extends React.Component {
   styles = (theme) => {
     const color = this.props.color || theme.Colors.PRIMARY;
 
-    return {
+    return _merge({}, {
```

Comment:
You can wrap this in an if checking for the prop so we aren't merging nothing if we aren't passing in any styles.

Reply:
@phantomxc Great idea. I have that updated and found a typo. Should be good to go. Thanks!